proficio_sandbox/dual_rehab.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * two_wams.cpp
+ *
+ *  Created on: Feb 15, 2011
+ *      Author: dc
+ */
+
+#include <iostream>
+
+#include <boost/thread.hpp>
+#include <boost/function.hpp>
+
+#include <barrett/exception.h>
+#include <barrett/units.h>
+#include <barrett/systems.h>
+#include <barrett/products/product_manager.h>
+
+
+using namespace barrett;
+
+
+boost::thread startWam(ProductManager& pm,
+		boost::function<void (ProductManager&, systems::Wam<3>&)> wt3);
+template <size_t DOF> void wamThread0(ProductManager& pm0, systems::Wam<DOF>& wam0);
+template <size_t DOF> void wamThread1(ProductManager& pm1, systems::Wam<DOF>& wam1);
+
+
+int main(int argc, char** argv) {
+	// Give us pretty stack-traces when things die
+	installExceptionHandler();
+
+	//ProductManager pm0("/etc/barrett/bus0/bus0.conf");
```

Comment:
This is the same example as dual_rehab_wam.cpp but  believe that this is not yet complete. We can modify this example and get it running. Can you please raise this as an issue, with enhancement tag and assign it to me?